admin/includes/cmb2-extra.php

```diff
@@ -24,6 +24,12 @@ function ds_cmb_render_multicheck_posttype( $field, $escaped_value, $object_id,
 			if ( in_array( $cpt, $values ) ) {
 				$args[ 'checked' ] = 'checked';
 			}
+
+			// @TODO Possibly find a solution if CMB2 trunk commit 1b3e251f8db429b3930e44ff43495838a9a39744
```

Comment:
Yeah, I guess that should work but I don't think it should be merged in just like that. I guess we would need to check CMB2 version number first. Furthermore, the issue is not relevant yet but it will be if/when CMB2 merge the current version of the trunk branch into master and release that. CMB2 2.2.* is fine for now.